src/Project/Project.types.ts

```diff
@@ -9,6 +9,7 @@ export type ProjectAttributes = {
   rows: number
   created_at: Date
   updated_at: Date
+  is_public: boolean
```

Comment:
nitpicking here, but can we move this above `created_at`? to keep it consistent. Same goes with `projectSchema`